templates/frontend/components/header.tpl

```diff
@@ -72,74 +72,16 @@
+						{load_navigationMenuArea name="primary" ulClass="pkp_navigation_primary"}
 
-							{* Primary navigation menu for current application *}
-							{include file="frontend/components/primaryNavMenu.tpl"}
-
-							{* Search form *}
-							{include file="frontend/components/searchForm_simple.tpl"}
-						</div>
-					</nav>
+						{* Search form *}
+						{include file="frontend/components/searchForm_simple.tpl"}
+					</div>
 				{/if}
 
-				<nav class="pkp_navigation_user_wrapper" id="navigationUserWrapper" aria-label="{translate|escape key="common.navigation.user"}">
-					<ul id="navigationUser" class="pkp_navigation_user pkp_nav_list">
-						{if $isUserLoggedIn}
-							<li class="profile {if $unreadNotificationCount} has_tasks{/if}">
-								<a href="{url router=$smarty.const.ROUTE_PAGE page="submissions"}">
-									{$loggedInUsername|escape}
-									<span class="task_count">
-										{$unreadNotificationCount}
-									</span>
-								</a>
-								<ul>
-									{if array_intersect(array(ROLE_ID_MANAGER, ROLE_ID_ASSISTANT, ROLE_ID_REVIEWER, ROLE_ID_AUTHOR), (array)$userRoles)}
-										<li>
-											<a href="{url router=$smarty.const.ROUTE_PAGE page="submissions"}">
-												{translate key="navigation.dashboard"}
-												<span class="task_count">
-													{$unreadNotificationCount}
-												</span>
-											</a>
-										</li>
-									{/if}
-									<li>
-										<a href="{url router=$smarty.const.ROUTE_PAGE page="user" op="profile"}">
-											{translate key="common.viewProfile"}
-										</a>
-									</li>
-									{if array_intersect(array(ROLE_ID_SITE_ADMIN), (array)$userRoles)}
-									<li>
-										<a href="{if $multipleContexts}{url router=$smarty.const.ROUTE_PAGE context="index" page="admin" op="index"}{else}{url router=$smarty.const.ROUTE_PAGE page="admin" op="index"}{/if}">
-											{translate key="navigation.admin"}
-										</a>
-									</li>
-									{/if}
-									<li>
-										<a href="{url router=$smarty.const.ROUTE_PAGE page="login" op="signOut"}">
-											{translate key="user.logOut"}
-										</a>
-									</li>
-									{if $isUserLoggedInAs}
-										<li>
-											<a href="{url router=$smarty.const.ROUTE_PAGE page="login" op="signOutAsUser"}">
-												{translate key="user.logOutAs"} {$loggedInUsername|escape}
-											</a>
-										</li>
-									{/if}
-								</ul>
-							</li>
-						{else}
-							{if !$disableUserReg}
-								<li><a href="{url router=$smarty.const.ROUTE_PAGE page="user" op="register"}">{translate key="navigation.register"}</a></li>
-							{/if}
-							<li><a href="{url router=$smarty.const.ROUTE_PAGE page="login"}">{translate key="navigation.login"}</a></li>
-						{/if}
-					</ul>
-				</nav><!-- .pkp_navigation_user_wrapper -->
-
+				{load_navigationMenuArea name="user" ulClass="pkp_navigation_user" navClass="pkp_navigation_user_wrapper" liClass="profile"}
```

Comment:
Restore the`<nav>` wrapper with it's `aria-label` here too.

Reply:
OK